content/tutorials/manage-the-rippled-server/install-rippled.md

```diff
@@ -46,59 +85,234 @@ This section assumes that you are using CentOS 7 or Red Hat Enterprise Linux 7.
 
         $ sudo systemctl start rippled.service
 
-## Installation on Ubuntu with alien
+5. Verify that `rippled` has started.
 
-This section assumes that you are using Ubuntu 15.04 or later.
+        $ sudo systemctl status rippled
 
-1. Install yum-utils and alien:
+For next steps, see [Postinstall](#postinstall) and [Additional Configuration](#additional-configuration).
 
-        $ sudo apt-get update
-        $ sudo apt-get install yum-utils alien
 
-2. Install the Ripple RPM repository:
+## Installation on macOS
 
-        $ sudo rpm -Uvh https://mirrors.ripple.com/ripple-repo-el7.rpm
+At this time, we don't recommend using the macOS platform for `rippled` production use. For production, consider using the [Ubuntu platform](#installation-on-ubuntu-with-alien), which has received the highest level of quality assurance and testing.
 
-3. Download the `rippled` software package:
+That said, macOS is suitable for many development and testing tasks. `rippled` has been tested for use with macOS High Sierra up to 10.13.
 
-        $ yumdownloader --enablerepo=ripple-stable --releasever=el7 rippled
+{% set n = cycler(* range(1,99)) %}
 
-4. Verify the signature on the `rippled` software package:
+{{n.next()}}. Install [Xcode](https://developer.apple.com/download/).
 
-        $ sudo rpm --import https://mirrors.ripple.com/rpm/RPM-GPG-KEY-ripple-release && rpm -K rippled*.rpm
+{{n.next()}}. Install Xcode command line tools.
 
-5. Install the `rippled` software package:
+        xcode-select --install
 
-        $ sudo alien -i --scripts rippled*.rpm && rm rippled*.rpm
+{{n.next()}}. Install [Homebrew](https://brew.sh/).
 
-6. Configure the `rippled` service to start on system boot:
+        ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
 
-        $ sudo systemctl enable rippled.service
+{{n.next()}}. Update Homebrew.
 
-7. Start the `rippled` service
+        brew update
+
+{{n.next()}}. Use Homebrew to install dependencies.
+
+        brew install git cmake pkg-config protobuf openssl ninja
+
+{{n.next()}}. Install Boost 1.67.0. Currently, we support Boost 1.67 only.
+
+        brew install boost@1.67
+
+{{n.next()}}. Ensure that your `BOOST_ROOT` environment variable points to the directory created by the Boost installation. To find your Boost install directory, use `brew info boost`. Put this environment variable in your `.bash_profile` file so it's automatically set when you log in. For example:
+
+        export BOOST_ROOT=/usr/local/Cellar/boost/1.67.0_1
```

Comment:
Thank you so much for taking the time to explain this convention to me. 👍  My work life is changed forever. I've updated this doc accordingly. It would be great to include it in the style guide.